src/camera/core.ts

```diff
@@ -21,7 +21,7 @@ export interface CameraCapability<T> {
   apply(value: T): Promise<void>;
 
   /** Returns current value of this capability. */
-  value(): T | null;
+  value(): T;
```

Comment:
`T | null` was supposed to indicate that value may not always be present, like for example when capability is not supported.

Is there a reason to prefer empty string over null?

Reply:
Same here